packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -484,16 +484,21 @@ const AuthAndAccountSetupRoutes = ({
         {...{ isSignedIn, serviceName }}
       />
 
-      <InlineTotpSetupContainer
-        path="/inline_totp_setup/*"
-        integration={integration as OAuthIntegration}
-        {...{ isSignedIn, serviceName, flowQueryParams }}
-      />
-      <InlineRecoverySetupContainer
-        path="/inline_recovery_setup/*"
-        integration={integration as OAuthIntegration}
-        {...{ isSignedIn, serviceName }}
-      />
+      {isOAuthIntegration(integration) && (
```

Comment:
Should this redirect if not an OAuthIntegration?

Reply:
Where are you think it would redirect to?